libtcmu.c

```diff
@@ -243,8 +243,13 @@ static int generic_handle_cmd(struct tcmu_device *dev,
 	size_t half = l / 2;
 	uint32_t cmp_offset;
 
-	cmd = cdb[0];
+	if (store->handle_cmd)
+		ret = store->handle_cmd(dev, tcmulib_cmd);
+
+	if (ret != TCMU_NOT_HANDLED)
```

Comment:
Initialize `ret` with `TCMU_NOT_HANDLED` in case `store->handle_cmd` is not exported by handlers.